benchto-driver/src/main/java/io/trino/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -179,6 +221,7 @@ private FinishRequest buildExecutionFinishedRequest(QueryExecutionResult executi
         if (executionResult.getPrestoQueryId().isPresent()) {
             requestBuilder.addAttribute("prestoQueryId", executionResult.getPrestoQueryId().get());
         }
+        requestBuilder.addAttribute("queryName", executionResult.getQueryExecution().getQueryName());
```

Comment:
Benchto should already have `queryName`, why we need this?

Reply:
Current queryName is a variable on benchmark level - it would make more sense to have is on attribute level. But it's a leftover from earlier version of code when I reported all executions - I will remove it.